docs/release_doc.md

```diff
@@ -0,0 +1,97 @@
+# Release documentation
+
+## preparation
+
+Create virtual env:
+
+```
+python3 -m venv .venv
+source .venv/bin/activate
+pip install -r release-requirements.txt
+```
+
+
+Create GitHub token, see documentation: [GH_TOKEN](https://python-semantic-release.readthedocs.io/en/latest/#github-gh-token).
+You can then store it in vars.env file like:
+```
+export GH_TOKEN=github_token_content
+```
+
+And before release do: `source vars.env`.
+
+Depends on settings of the repository for protected branches but we have
+protected branch which requires PR to push to main branch, hence we use
+`--no-push` option.
+
+In my case, the origin (this) repository remote is called `origin` and my fork
+is called pbalogh.
+
+Checkout to main branch after merging all PRs you would like to have included
+in the release.
+`git checkout main`
+
+And **rebase** on latest changes: `git fetch --all && git rebase origin/main`
+Or you you can even hard reset: `git reset --hard origin/main`
+
+**Do release**
+
+```shell
+$ semantic-release version --patch --no-push
+```
+
+This will create new commit and tag for you:
+```shell
+$ git log
+
+commit 295070509f011be59652e6bbc413546aa65ab82d (tag: v0.0.2)
+Author: semantic-release <semantic-release>
+Date:   Tue Nov 21 19:12:43 2023 +0530
+
+    0.0.2
+
+    Automatically generated by python-semantic-release
```

Comment:
can we update with below so that we aligned all example showed with v.0.0.1

```commit ae259e096e27c598ca0787ad090220c21f82b0a4 (tag: v0.0.1, module_for_remote_connection)
Author: semantic-release <semantic-release>
Date:   Fri Nov 3 11:23:54 2023 +0100

    0.0.1
    
    Automatically generated by python-semantic-release
```

Reply:
done